drv/stm32h7-update-server/src/main.rs

```diff
@@ -174,8 +174,8 @@ impl<'a> ServerImpl<'a> {
         // can't define them as `const` values.
         //
         // SAFETY: these are symbols populated by the linker.
-        let bank_addr = unsafe { __REGION_BANK2_BASE.as_ptr() } as usize;
-        let bank_end = unsafe { __REGION_BANK2_END.as_ptr() } as usize;
+        let bank_addr = &raw const __REGION_BANK2_BASE as usize;
```

Comment:
Do we still need the `SAFETY` comments above if this is no longer unsafe? (also wooo less unsafe 8) )